<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DamageReport</class>
 <widget class="QWidget" name="DamageReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Damage Report</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Supervisor*:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>* Required Field</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Package ID*:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Description*:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Employee*:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="trailerNumText"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="descriptionText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Trailer Number*:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Date*:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="employeeText"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="supervisorText"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="dateText"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="packageIDCombo"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>trailerNumText</tabstop>
  <tabstop>descriptionText</tabstop>
  <tabstop>employeeText</tabstop>
  <tabstop>supervisorText</tabstop>
  <tabstop>dateText</tabstop>
  <tabstop>submitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
